environments/production/invoke.yml

```diff
@@ -18,6 +18,7 @@ elastic_search:
   protocol: "https"
   is_aws: true
   verify_certs: true
+  enable_decompound_analyzers: true
```

Comment:
ik zag verderop iets over dat AWS dit niet goed ondersteund. Kan het dan wel aan voor productie?

Reply:
Ja, productie heeft nergens last van. Er staat een support ticket open voor acceptance. AWS is er volgens eigen zeggen mee bezig, maar we wilden niet langer wachten met dingen testen op acceptance. Vandaar de splitsing. Verre van ideaal, maar we gaan het voorzichtig deployen.